src/sentry/static/sentry/app/views/organizationIncidents/status.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'react-emotion';
+
+import InlineSvg from 'app/components/inlineSvg';
+import SentryTypes from 'app/sentryTypes';
+import space from 'app/styles/space';
+
+import {isResolved} from './utils';
+
+export default class Status extends React.Component {
+  static propTypes = {
+    incident: SentryTypes.Incident.isRequired,
+  };
+
+  render() {
+    const isIncidentResolved = isResolved(this.props.incident);
+
+    const icon = isIncidentResolved ? 'icon-circle-check' : 'icon-circle-exclamation';
+    const text = isIncidentResolved ? 'Resolved' : 'Active';
+
+    return (
+      <Container>
```

Comment:
I think for these smaller, re-used components, we should allow them to be styled using emotion by either passing in `className` to `Container`, or export the styled component instead.

Reply:
Do we need views that aren't considered shared components (i.e. in the views folder and not used outside incidents) to implement this behavior even when it's not used? My vote is no. It's likely that we will actually need this here for the subscribe status button, but I'd prefer to add when needed.

Reply:
I would say no - views are generally not shared.